README.md

```diff
@@ -1 +1,84 @@
-# ride-share-two
\ No newline at end of file
+# Ride Share
+Remember the ride share exercise we did with designing and creating a system to track the ride share data from a CSV file? We did a lot of great work on this exercise in creating arrays and hashes of data, but we've learned a lot since we did that exercise! Now, we're going to use our understanding of classes, methods and attributes to create an object-oriented implementation of our ride share system.
+
+## Learning Goals
+Reinforce and practice all of the Ruby and programming concepts we've covered in class so far:
+- Creating and instantiating classes with attributes
+- Creating class and instance methods within our classes
+- Writing pseudocode and creating tests to drive the creation of our code
+- Importing data from a CSV file
+
+## Primary Requirements
+Create a system that will manage information related to ride sharing. The system should have a way to keep track of drivers, riders and the trips that the drivers take with their riders.
+
+
+#### Driver
+- Each driver should have information uniquely identifying them like ID, name, license plate and vehicle identification number
+- Each vehicle identification number should abide by a specific pattern or length to ensure it is a valid vehicle identification number
```

Comment:
I think it would be helpful for the students to see that the requirement was for a VIN to be exactly X characters. Then maybe we could add an optional for validating it more strictly.